src/pretalx/submission/models/type.py

```diff
@@ -75,6 +75,13 @@ def __str__(self) -> str:
         """Used in choice drop downs."""
         if not self.default_duration:
             return str(self.name)
+
+        if hasattr(self, "event") and hasattr(self.event, "cfp"):
+            if not self.event.cfp.fields.get("duration", {}).get(
```

Comment:
Isn’t this logic the wrong way around? Like this, you are only returning the name if the duration field is **not** required, which does not correspond with the original requirements, if I'm not mistaken.